### DISCUSSION TITLE
Some pics, anyone…

### DISCUSSION THREAD

**Discussion by casandra9** [ataradov/usb-sniffer#27] [Show and tell]
Hi,
Please, could someone who completed his unit post a picture of the front and back with components soldered straight from top view, high resolution.
I am in the process of soldering my device and this would be extremely helpful.

Thank you very much for this wonderful project!

**Comment by ataradov** [comment]
I don't have the best lighting setup, but I think this should be good enough for assembly purposes. I'm not sure how this is helpful anyway, none of the passives have any markings, so you need to reference the CAD files anyway.
![back](https://github.com/ataradov/usb-sniffer/assets/4887933/4e1427b6-8a80-4424-8a0b-227166124a38)
![front](https://github.com/ataradov/usb-sniffer/assets/4887933/b4302dea-c9b5-4d55-a8c8-8c6477ff1b8a)

Note that there are a couple mods on the capture side DM/DP. Those are just for some unrelated tests, you don't need to make them.

**Comment by casandra9** [comment]
Thank you a lot!

Greetings from BC, Thomas

casandra
***@***.***
